lib/pantry.rb

```diff
@@ -0,0 +1,39 @@
+require './lib/recipe'
+
+class Pantry
+  attr_reader :stock
+
+  def initialize
+    @stock = {}
+  end
+
+  def stock_check(ingredient)
+    if @stock[ingredient] == nil
+      0
+    else
+      @stock[ingredient]
+    end
+  end
+
+  def restock(ingredient, amount)
+    if @stock.keys.include?(ingredient) == false
```

Comment:
I think this bit could have been simplified a bit if we used a default value for the hash.